index.html

```diff
@@ -297,6 +310,43 @@ <h2>The VC HTTP API</h2>
       <h3>Authorization</h3>
       <p>
       </p>
+
+      <p class="issue" title="Apply group resolution">
+Convert the following group resolution into specification text:<br><br>
+<a href="https://w3c-ccg.github.io/meetings/2021-07-13-vchttpapi/#resolution-1">
+RESOLUTION</a>: The VC HTTP API work item group will separate GNAP from OAuth2
+until it is clear how much extra work GNAP would add within the scope of the
+specification.
+      </p>
+
+      <p class="issue" title="Apply group resolution">
+Convert the following group resolution into specification text:<br><br>
+<a href="https://w3c-ccg.github.io/meetings/2021-07-13-vchttpapi/#resolution-2">
+RESOLUTION</a>: One of the authorization mechanisms defined for the VC-HTTP-API
+MUST be OAuth 2 Bearer tokens.
+      </p>
+
+      <p class="issue" title="Apply group resolution">
+Convert the following group resolution into specification text:<br><br>
+<a href="https://w3c-ccg.github.io/meetings/2021-07-13-vchttpapi/#resolution-3">
+RESOLUTION</a>: How a VC HTTP API server validates an authorization token is out
+of scope.
+      </p>
+
+      <p class="issue" title="Apply group resolution">
+Convert the following group resolution into specification text:<br><br>
+<a href="https://w3c-ccg.github.io/meetings/2021-07-13-vchttpapi/#resolution-4">
+RESOLUTION</a>: One of the authorization protocols that will be defined for use
+in the VC-HTTP-API MUST be OAuth 2 Client Credentials.
+      </p>
+
+      <p class="issue" title="Apply group resolution">
+Convert the following group resolution into specification text:<br><br>
+<a href="https://w3c-ccg.github.io/meetings/2021-07-13-vchttpapi/#resolution-5">
+RESOLUTION</a>: One of the authorization mechanisms defined for the VC HTTP API
+MUST be GNAP key-bound access tokens.
```

Comment:
```suggestion
The group has decided to delay the discussion of implementing GNAP key-bound access tokens as one of the authorization mechanisms for the VC HTTP API until January 2022. The group might decide to take the topic up before that date if there is agreement to do so.
```

@jandrieu, @agropper, @jricher -- would any of you object to the text above?